src/Storage/Adapter/Runtime.php

```diff
@@ -26,16 +24,16 @@ class Runtime implements StorageInterface
 
     /**
      * Initializes a new instance of this class.
+     *
+     * @param array $data
      */
-    public function __construct()
+    public function __construct(array $data = [])
```

Comment:
I added this so we can easily use it for tests where you need certain data to be already stored.